exercises/loops/problem.en.md

```diff
@@ -42,7 +42,7 @@ done
 
 ### `while` loop
 
-The `while` loop tests a condition and loops a sequence of commands until that condition is _true_. A condition is nothing more than primary is used in `if..then` conditions. So a working example with `while` loop might look like this:
```

Comment:
Why _until_?

`while` performs command till that condition is `true`, isn't it?